ipamd/ipamd.go

```diff
@@ -208,7 +253,11 @@ func (c *IPAMContext) nodeInit() error {
 
 	log.Debugf("Start node init")
 
-	instanceMaxENIs, _ := c.awsClient.GetENILimit()
+	instanceMaxENIs, err := c.awsClient.GetENILimit()
+	if err != nil {
+		log.Errorf("Failed to get ENI limit: %s")
```

Comment:
Should we not error out in case the limits cannot be determined

Reply:
I agree we should handle unknown instance types in a better way, but this change was just to keep it consistent with line 576 were we do the same thing.